tests/transports-integrations/tests/utils/common.py

```
@@ -0,0 +1,412 @@
+]
+
+
+# Helper Functions
+def mock_tool_response(tool_name: str, args: Dict[str, Any]) -> str:
+    """Generate mock responses for tool calls"""
+    if tool_name == "get_weather":
+        location = args.get("location", "Unknown")
+        unit = args.get("unit", "fahrenheit")
+        return f"The weather in {location} is 72°{'F' if unit == 'fahrenheit' else 'C'} and sunny."
+
+    elif tool_name == "calculate":
```

Ruff (0.11.9)

Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)